core/src/main/scala/cats/data/NonEmptyMapImpl.scala

```diff
@@ -78,6 +78,28 @@ sealed class NonEmptyMapOps[K, A](val value: NonEmptyMap[K, A]) {
   def map[B](f: A => B): NonEmptyMap[K, B] =
     NonEmptyMapImpl.create(Functor[SortedMap[K, *]].map(toSortedMap)(f))
 
+  /**
+   * Applies f to all the keys leaving elements unchanged.
+   */
+  def leftMap[L](f: K => L)(implicit orderL: Order[L]): NonEmptyMap[L, A] = {
```

Comment:
I think `mapKeys` might be a better name here.

Reply:
updated